### PR TITLE
Add support for django 1.11

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

**Comment by jlward** [comment]
Dropping support for python 2.6 will fix the tests on travis.

File: tox.ini [hunk 1]
```diff
@@ -17,18 +17,16 @@ commands =
...
     django1.8.X: django==1.8,<1.9
+    django1.11.X: django==1.11,<1.12
     # Something odd going on with travis won't let those tests pass. Ignore
     # them for now.
     # selenium==2.47
-    py{26,27}: CherryPy==3.2.0
-    py{34}: CherryPy==5.0.1
-    south: south
+    py27: CherryPy==3.2.0
+    py34: CherryPy==5.0.1
```
Comment (kylegibson) [inline tox.ini]: Since we're commenting out selenium, why do we need to include cherrypy?

File: nosedjango/nosedjango.py [hunk 2]
```diff
@@ -289,7 +289,10 @@ def begin(self):
             self.call_plugins_method(
                 'beforeTestSetup', settings, setup_test_environment,
                 connection)
-        setup_test_environment()
+        try:
+            setup_test_environment()
+        except RuntimeError:  # Django 1.11 + multiprocess this happens.
+            pass
```
Comment (kylegibson) [inline nosedjango/nosedjango.py]: So we can just catch the exception and everything works anyway?

File: tox.ini (hunk 1 above)
Comment (jlward) [reply tox.ini]: This is needed because we are testing the cherrypy plugin.

File: nosedjango/nosedjango.py (hunk 2 above)
Comment (jlward) [reply nosedjango/nosedjango.py]: Yea. The exception had to do with the test env already being called. So no reason to call it multiple times.

**Comment by kylegibson** [comment]
Looks good :+1:

**Pull request merged**